tests/test_open_images_format.py

```diff
@@ -8,12 +8,82 @@
 
 import numpy as np
 
-from datumaro.plugins.open_images_format import OpenImagesImporter
-from datumaro.util.test_utils import compare_datasets_strict
+from datumaro.plugins.open_images_format import OpenImagesConverter, OpenImagesImporter
+from datumaro.util.image import Image
+from datumaro.util.test_utils import TestDir, compare_datasets_strict
 from datumaro.components.extractor import AnnotationType, DatasetItem, Label, LabelCategories
 from datumaro.components.dataset import Dataset
 from tests.requirements import Requirements, mark_requirement
 
+class OpenImagesFormatTest(TestCase):
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_save_and_load(self):
+        source_dataset = Dataset.from_iterable(
+            [
+                DatasetItem(id='a', subset='train',
+                    annotations=[Label(0, attributes={'score': 0.7})]
+                ),
+                DatasetItem(id='b', subset='train', image=np.zeros((8, 8, 3)),
+                    annotations=[Label(1), Label(2, attributes={'score': 0})]
+                ),
+            ],
+            categories={
+                AnnotationType.label: LabelCategories.from_iterable([
+                    '/m/0',
+                    ('/m/1', '/m/0'),
+                    '/m/2',
+                ]),
+            },
+        )
+
+        with TestDir() as test_dir:
+            OpenImagesConverter.convert(source_dataset, test_dir,
+                save_images=True)
+
+            parsed_dataset = Dataset.import_from(test_dir, 'open_images')
+
+            # the converter assumes that labels without a score have a score of 100%
+            source_dataset.get('b', subset='train').annotations[0].attributes['score'] = 1
+
+            compare_datasets_strict(self, source_dataset, parsed_dataset)
```

Comment:
`compare_datasets_strict` is supposed to check for exact equality (including item order). Probably, it is not required for most formats. I suggest to use a more relaxed version - `compare_datasets`.

Reply:
But the Open Images format _does_ preserve the item order (since it saves image metadata in a CSV file), so why not check it?

Reply:
I understand, but such requirement is not supposed or enforced by anything in the project now. So I consider it is too fragile and restrictive for now. `datumaro` is the only format, which guarantees this now, and I don't think we need to guarantee this for other formats (even though they are support it and formats implement it), unless we implement some kind of item order API etc.

Reply:
OK, updated (I also took the liberty of updating the old tests).